public/data/workflows/meeting.json

```diff
@@ -1,127 +0,0 @@
-{
```

Comment:
This can become a property on workflow - a serialized version of the workflow schema. serializing/deserializing can happen on the backend or frontend, and will also neccesitate a switch to uuids.

Reply:
is the behavior you're describing implemented?